Caused by: com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource compilation failed
/home/runner/work/openfoodfacts-androidapp/openfoodfacts-androidapp/app/src/opf/res/values-tr/strings.xml:46:5-81: AAPT: error: unescaped apostrophe in string
    
/home/runner/work/openfoodfacts-androidapp/openfoodfacts-androidapp/app/src/opf/res/values-tr/strings.xml:46:5-81: AAPT: error: not a valid string.
    
/home/runner/work/openfoodfacts-androidapp/openfoodfacts-androidapp/app/build/intermediates/incremental/mergeOpfDebugResources/merged.dir/values-tr/values-tr.xml: AAPT: error: file failed to compile.

--- a/app/src/opf/res/values-tr/strings.xml
+++ b/app/src/opf/res/values-tr/strings.xml
@@ -43,7 +43,7 @@
     <string name="add_nutrient_category_prompt_text">Ürünü tamamlamaya yardım etmek için bir kategori ve daha fazla ayrıntı ekleyin</string>
     <string name="txt_eaten_products">Sahip olduğum ürünler</string>
     <string name="txt_products_to_buy">Satın alınacak ürünler</string>
-    <string name="intent_filter_share_label">Open Products Facts'e ekle</string>
+    <string name="intent_filter_share_label">Open Products Facts\'e ekle</string>
     <string name="txt_info_eaten_products">Bu listeye ürün eklemeniz ürünleri tarayarak, ürün kartını yukarıya sürükleyip \"Satın alınacak ürünler\"e tıklayarak mümkün olacaktır.
 Aynı zamanda bu listeye ürünleri arayarak ve ürün sayfasının sağ üstündeki + butonuna tıklayarak ürünleri ekleyebilirsiniz.</string>
     <string name="OFFlogo">Open Beauty Facts amblemi</string>

